opennms-container/minion/plugins.sh

```diff
@@ -1,13 +1,14 @@
 #!/bin/bash
 
+set -euo pipefail
```

Comment:
```suggestion
set -euo pipefail
trap 's=$?; echo "$0: Error on line "$LINENO": $BASH_COMMAND"; exit $s' ERR
```

Whenever you use `set -e`, I **strongly** recommend something like this trap, otherwise you will get **silent failures** if a command returns a non-zero exit status but doesn't print anything, and at some point, you will really wish you know why the script exited. It might. not matter for **this** script, but you never know when someone will copy this script to create a new script.

Even if you still get some error output, if you use that tool a number of times, you might not know *which* instance of it failed. The trap will let you know which line (usually; sometimes it is buggy, but most of the time it's right or will point you at the right command).